internal/sourcecode/scan.go

```diff
@@ -202,6 +202,11 @@ func configureRails(sourceDir string) (*SourceInfo, error) {
 				Value: string(masterKey),
 			},
 		}
+
+		s.Env = map[string]string{
+			"RAILS_ENV":           "production",
+			"RAILS_LOG_TO_STDOUT": "true",
+		}
```

Comment:
@jsierles Why is this only included if `config/master.key` is set? They seem unrelated - should they be outside the conditional?

Reply:
Good catch! I pushed a fix.